.github/workflows/checker.yml

```diff
@@ -0,0 +1,76 @@
+name: Checker
```

Comment:
I'm not a big fan of the `checker` word, doesn't feel natural. We need to be more explicit about what this workflow and the rust feature behind it are actually doing.

Reply:
Yes, I was implying changing also the feature itself, as checker doesn't feel explicit nor natural (in the sense of the English language)

Reply:
Do you have any suggestion of another name?

Reply:
If we're creating a certificate and passing it to a TCE network that we run just for that, it really resembles e2e tests to me (we have a debate on this with @hadjiszs :p), as we're using the API of the present repository and verify the rest of the code works as expected (i.e., delivery is working).

We're essentially doing the same thing on k8s, with cert-spammer.

>I want to be able to add more check to debug the broadcast

I don't think adding more checks is making this less e2e, probably the opposite.

Reply:
My comment was really about the feature as a whole. If we're not changing the feature name, no need to change the workflow.

>(if not misleading as not fully for e2e)

I'm starting to understand I'm missing something about this feature. Looking at the origin PR (#185—is it normal btw that the two PRs have the same title?), I read this:

>This feature is really not aiming to be used in production and should only be used in e2e.

We might want to update descriptions to make things consistent across PRs.

Can we just detail here which use cases for `checker` to be used are not e2e related?